sdk/ml/azure-ai-ml/tests/pipeline_job/unittests/test_pipeline_job_entity.py

```diff
@@ -2034,3 +2034,10 @@ def empty_value_pipeline(integer: int, boolean: bool, number: float, str_param:
         rest_obj = pipeline._to_rest_object()
         expect_resource = {"instance_count": "${{parent.inputs.integer}}", "shm_size": "${{parent.inputs.shm_size}}"}
         assert rest_obj.properties.jobs["component"]["resources"] == expect_resource
+
+    def test_pipeline_job_with_queue_settings(self) -> None:
+        yaml_path = "./tests/test_configs/pipeline_jobs/serverless_compute/pipeline_with_queue_settings.yml"
+        pipeline_job = load_job(yaml_path)
+        rest_obj = pipeline_job._to_rest_object()
+        expected_queue_settings = {"job_tier": "Standard", "priority": 2}
```

Comment:
add an e2e test? we can skip if that does not work yet.

job_tier could be tested with serverless compute.
"job_tier": & "priority" should be tested with singularity case

Reply:
Sure, a submit & cancel E2E test should pass currently. Let me update this for `job_tier`, as we target serverless compute in this PR.

Reply:
Updated this UT & added one E2E test, may take a look.